build/onos-gui/nginx.conf

```diff
@@ -45,13 +45,13 @@ http {
           root /usr/share/nginx/html;
         }
         location /onos-config/ {
-            proxy_pass http://onos-config-envoy:8080/;
+            proxy_pass http://localhost:8081/;
```

Comment:
@SeanCondon 
Is the proxy pass refers to the k8s service name? I got gui running based on the previous setup and in the previous setup this field refers to name of envoy service in k8s. Perhaps it would be better to pick onos-envoy.

Reply:
I just noticed you combined the two in one so that means we just need to deploy gui image now. However, it is possible to deploy envoy separately now as well.

Reply:
no it doesn't - the service name is onos-gui.
I could use onos-gui here but it would only be a loopback on itself
The envoy container is in the same pod as the nginx, so i can just refer to it by localhost